src/hooks/usePaginate.ts

```diff
@@ -0,0 +1,14 @@
+import { useState } from "react"
+
+export const usePaginate = (
+  initialPage = 0
```

Comment:
Hmm why will there be a case when initial page is not 0 ah?

Reply:
lets say that we want to store current page view between loads etc. 

it's a case where it's actually more effort to remove (and less functionality) than to just have it there